dss/events/chunkedtask/aws.py

```diff
@@ -12,23 +18,98 @@
 logger.setLevel(logging.INFO)
 
 
-def dispatch(context, payload):
+def schedule_task(client_name: str, state: typing.Any):
+    task_id = str(uuid.uuid4())
+
+    payload = {
+        awsconstants.CLIENT_KEY: client_name,
+        awsconstants.REQUEST_VERSION_KEY: awsconstants.CURRENT_VERSION,
+        awsconstants.TASK_ID_KEY: task_id,
+        awsconstants.STATE_KEY: state,
+    }
+
+    sts_client = boto3.client('sts')
+    accountid = sts_client.get_caller_identity()['Account']
+
+    sns_client = boto3.client('sns')
+    region = os.environ['AWS_DEFAULT_REGION']
```

Comment:
I think `boto3.Session().region_name` is preferable. The env var is not always set, but that snippet will always follow the full boto3 config ingestion protocol.